configs/13.0/specs/release.spec

```diff
@@ -27,15 +30,15 @@
 # The date when this Product goes out of support. A missing value indicates
 # that there will be no EOL, while an empty/invalid value indicates that
 # there will be an EOL date, but it's not yet known.
-%define         productendoflife ""
-%define         productendoflifel_pcescaped %{productendoflife}
+%define         productendoflife __AT_END_OF_LIFE__
+%define         productendoflife_pcescaped __AT_END_OF_LIFE_ESCAPED__
 
 %define         producturlbugtracker https://github.com/advancetoolchain/advance-toolchain/issues/
 %define         producturlbugtracker_pcescaped https%3A%2F%2Fgithub.com%2Fadvancetoolchain%2Fadvance%2Dtoolchain%2Fissues%2F
 
 # ----------------------------------------------------------------------
 
-Name:           %{product}-release
+Name:           %{package}-release
 Summary:        Advance Toolchain
 Version:        %{productversion}
 Release:        %{productrelease}
```

Comment:
AFAIU, the way this is written could allow for a `-release` package from version 15.0 be installed without having an AT 15.0 installed.
Should we add a line around line 60 with the following?
```
Suggests: advance-toolchain-%{at_major}-runtime
```

This won't enforce the version, but would help the resolver to get the latest one.

Reply:
Done